src/helpers/query.helper.ts

```diff
@@ -51,4 +53,17 @@ export class QueryHelper {
             },
     };
   }
+
+  aliasFactory<E extends Record<string, any>>(
```

Comment:
```suggestion
  aliasFactory<E extends Record<string, unknown>>(
```

any 타입보다는 unknown을 사용하는게 타입추론에 도움이 됩니다.